SwiftWisdom/Core/UIKit/UIApplication+Extensions.swift

```diff
@@ -13,10 +13,14 @@ enum OpenSettingsError: Error {
 }
 
 extension UIApplication {
+
+    @available(iOS, unavailable, message: "Call UIApplication.open(_:) directly")
```

Comment:
Can this give a more exact hint of a change? My suggestion:
`"Call open(_:) directly using UIApplication.openSettingsURLString"`

Reply:
Good point, should I do that, and / or tell the client to "see the source code for more information" (since this version leaves the original code intact, just uncompiled)?